accounts/addproofer.js

```diff
@@ -0,0 +1,29 @@
+/*global $*/
```

Comment:
Can you run this through eslint too? IIRC we are using 4-space indents instead of 2- and that would validate this.

Reply:
We have an outstanding task for getting that into our CI/CD too.